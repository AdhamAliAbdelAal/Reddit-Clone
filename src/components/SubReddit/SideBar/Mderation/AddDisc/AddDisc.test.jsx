import '@testing-library/jest-dom';
import renderer from 'react-test-renderer';
import {
  fireEvent,
  render, screen,
} from '@testing-library/react';
import AddSector from './Add';

it('open input to add disc', async () => {
  // window.history.pushState({}, '', '/Login');
  render(<AddSector />);
  const add = screen.getByTestId('add');
  fireEvent.click(add);
  const input = screen.getByTestId('input');
  expect(input).toBeInTheDocument();
});

// test snapshot
test('test snapshot', async () => {
  const tree = renderer.create(<AddSector />).toJSON();
  expect(tree).toMatchSnapshot();
});
