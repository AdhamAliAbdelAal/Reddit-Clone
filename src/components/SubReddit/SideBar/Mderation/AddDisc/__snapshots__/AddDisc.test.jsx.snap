// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test snapshot 1`] = `
<h2
  className="css-2v81eu"
>
  <div
    className="css-1pmeph4"
    onClick={[Function]}
    onTouchEnd={[Function]}
  >
    <div
      className="MuiBox-root css-k008qs"
      data-testid="add"
      onClick={[Function]}
    >
      <div
        className="MuiBox-root css-156oqa0"
      >
         
        Ahmed
      </div>
       
      <svg
        aria-hidden={true}
        className="MuiSvgIcon-root MuiSvgIcon-colorPrimary MuiSvgIcon-fontSizeMedium css-ci5apu-MuiSvgIcon-root"
        data-testid="ModeEditOutlineOutlinedIcon"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M3 21h3.75L17.81 9.94l-3.75-3.75L3 17.25V21zm2-2.92 9.06-9.06.92.92L5.92 19H5v-.92zM18.37 3.29a.9959.9959 0 0 0-1.41 0l-1.83 1.83 3.75 3.75 1.83-1.83c.39-.39.39-1.02 0-1.41l-2.34-2.34z"
        />
      </svg>
    </div>
  </div>
</h2>
`;
